src/Token.react.js

```diff
@@ -42,7 +42,7 @@ const Token = React.createClass({
 
   _renderRemoveableToken() {
     return (
-      <button
+        <div 
```

Comment:
Did you consider using an `<a>` instead? Given that the removable token is meant to be interactive, an `<a>` might reflect that a bit better.